samples/listenForMessages.js

```diff
@@ -28,6 +28,8 @@
 //   usage: node listenForMessages.js <subscription-name> [timeout-in-seconds]
 
 function main(subscriptionName = 'YOUR_SUBSCRIPTION_NAME', timeout = 60) {
+  timeout = Number(timeout);
```

Comment:
Is this being added here for the sake of argument validation?  Or do we need to do the coercion here for some reason?  Asking because we don't tend to do argument validation in the sample code, and it's one of those things I think we should probably avoid.

Reply:
Yeah, basically if you pass a timeout on the command line, it will be a string. I honestly try my hardest not to indulge in JavaScript string coercion, but this is the only place it's used. Will that work?

```
    setTimeout(() => {
      subscription.removeListener('message', messageHandler);
      console.log(`${messageCount} message(s) received.`);
    }, timeout * 1000);
```

Reply:
It turns out that "50"*1000 is indeed a number, but it just feels so gross. I was willing to let that go for sample simplicity, but there are some places where I think it will genuinely cause issues, like samples where the value is passed directly into JSON that is sent elsewhere. So I'm back to thinking this normalization is okay, as long as it happens outside the region tags. The samples on the site will still look clean, and the .js files are already kinda gooped with exception handling and such, so I don't feel too bad about that.